docker-compose.yml

```
@@ -78,7 +78,7 @@ services:
       context: .
       dockerfile: ./docker/mailhog/Dockerfile
     container_name: mailhog_theater
-    command: ["/bin/bash", "-c", "/commands/setup_mailhog_auth.sh && ~/go/bin/MailHog"]
+    command: ["/bin/bash", "-c", "/commands/setup_mailhog_auth.sh && /go/bin/MailHog"]
```

The mailhog service command uses `/bin/bash -c "/commands/setup_mailhog_auth.sh && /go/bin/MailHog"`. Make sure the Dockerfile used to build this image installs bash and places your scripts at `/commands/...`. If the image doesn't include bash, this will fail at container start. Alternatives:

- Use `/bin/sh -c` if your scripts are POSIX-compatible and `sh` is available, or
- Ensure the Dockerfile installs `bash` and sets up the `/commands` path.

This is important because the service is built locally (not a base MailHog image) and the command will fail silently if the shell is missing.